habitat_baselines/config/test/ppo_imagenav_test.yaml

```diff
@@ -0,0 +1,33 @@
+BASE_TASK_CONFIG_PATH: "configs/tasks/imagenav.yaml"
```

Comment:
Can you add a version of this with the hyper-parameters used for training?

Reply:
Hparams used for training are in habitat_baselines/config/imagenav/ddppo_imagenav_gibson.yaml